appdatabase/migrations/sql/1635942153_add_telemetry_server_url_to_settings.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE settings ADD COLUMN telemetry_server_url VARCHAR NOT NULL DEFAULT "";
```

Comment:
I think this might be problematic on upgrades, as the row will have a NULL value (defaults are not applied retrospectively).
Definitely you will have issues on upgrade with the code below, as it will try to read a `NULL` into a non-nullable `string`.
Generally you have 2 options, either you use a `sql.NullString` and check if it's valid when scanning the row, or you can just make sure it's always there, by setting it in the migration, so add a line:
`UPDATE settings SET telemetry_server_url = ""`

That way you are guaranteed that it's not null on existing clients.

Reply:
Good point, I added the update in the migration